### PR TITLE
Check dbg! macro in tidy_test

### PR DESCRIPTION
**Pull request by edwin0cheng**
Same as `check_todo` but for dbg! macro

r? @matklad

### CODE REVIEW

File: xtask/tests/tidy.rs [hunk 1]
```diff
@@ -212,7 +212,7 @@ Zlib OR Apache-2.0 OR MIT
 fn check_todo(path: &Path, text: &str) {
     let need_todo = &[
         // This file itself obviously needs to use todo (<- like this!).
-        "tests/cli.rs",
+        "tests/tidy.rs",
```
Comment (edwin0cheng) [inline xtask/tests/tidy.rs:215]: Seem it worked due to the test of `text.contains("${0:todo!()}")`  :)

File: xtask/tests/tidy.rs [hunk 2]
```diff
@@ -240,6 +240,32 @@ fn check_todo(path: &Path, text: &str) {
     }
 }
 
+fn check_dbg(path: &Path, text: &str) {
+    let need_dbg = &[
+        // This file itself obviously needs to use todo (<- like this!).
```
Comment (jonas-schievink) [inline xtask/tests/tidy.rs:245]:
```suggestion
        // This file itself obviously needs to use dbg.
```

Comment (edwin0cheng) [reply xtask/tests/tidy.rs]: Fixed

**Pull request merged**